samples/fb-composition-layers-sample/export_presets.cfg

```diff
@@ -0,0 +1,227 @@
+[preset.0]
+
+name="Quest"
+platform="Android"
+runnable=true
+advanced_options=true
+dedicated_server=false
+custom_features=""
+export_filter="all_resources"
+include_filter=""
+exclude_filter=""
+export_path="../builds/FbPassthroughSample.apk"
```

Comment:
Looks like a leftover from the passthrough sample PR :)

Reply:
Good catch, updated!